lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -43,22 +43,19 @@ const INITIAL_DIALOG_STATE = {
 };
 
 const CanvasNoFiles = (
-  <Fragment>
-    <div className="FileList__no-files">
-      There are no available PDF files in the current course.
-    </div>
+  <div className="FileList__no-files">
     <p>
-      Please see the Canvas help article for{' '}
+      There are no PDFs in the course.{' '}
       <a
         href="https://community.canvaslms.com/t5/Instructor-Guide/How-do-I-upload-a-file-to-a-course/ta-p/618"
         target="_blank"
         rel="noreferrer"
       >
-        uploading files to a course
+        Updload some files to the course
```

Comment:
Typo: "Upload"
Suggest also changing "the" to "this" here:

"Upload some files to this course"